src/gd_kotlin.cpp

```diff
@@ -64,6 +64,20 @@ void load_classes_hook(JNIEnv* p_env, jobject p_this, jobjectArray p_classes) {
     classes.delete_local_ref(env);
 }
 
+void register_src_dirs_hook(JNIEnv* p_env, jobject p_this, jobjectArray p_src_dirs) {
+    jni::Env env(p_env);
+    jni::JObjectArray src_dirs_raw{jni::JObjectArray(p_src_dirs)};
+    Vector<String> src_dirs{};
```

Comment:
This can be written this way to get code easier to read:
```cpp
Vector<String> src_dirs;
```
This will call default constructor.